JetNews/app/src/main/java/com/example/jetnews/ui/article/ArticleScreen.kt

```diff
@@ -72,18 +73,18 @@ fun ArticleScreen(postId: String) {
                 title = {
                     Text(
                         text = "Published in: ${post.publication?.name}",
-                        style = MaterialTheme.typography().subtitle2
+                        style = MaterialTheme.typography.subtitle2.copy(color = contentColor())
                     )
                 },
                 navigationIcon = {
-                    VectorImageButton(R.drawable.ic_back) {
-                        navigateTo(Screen.Home)
+                    IconButton(onClick = { navigateTo(Screen.Home) }) {
+                        Icon(vectorResource(R.drawable.ic_back))
                     }
                 }
             )
         },
         bodyContent = { modifier ->
-            PostContent(modifier = modifier, post = post)
+            PostContent(post, modifier)
```

Comment:
Isn't it convention that `modifier` is the first param (on `PostContent`)?

Reply:
Convention is that modifier is the first default argument.

Reply:
Resolving per offline convo.